### DISCUSSION TITLE
Possible new features (brainstorm)

### DISCUSSION THREAD

**Discussion by nikosarcevic** [nikosarcevic/CosmOracle#67] [General]
- [ ] Distance modulus
- [ ] ~~Export py plot script~~
- [ ] #45
- [ ] (matter) power spectrum?

**Comment by lonbar** [comment]
Export py script is discussed in #13 and #16.

**Comment by nikosarcevic** [comment]
I think we agreed not to go with py plot code export?

**Comment by lonbar** [comment]
I'm confused. What exactly is the scope of export py plot script here that is not part of these other issues?

**Comment by nikosarcevic** [comment]
it is the same issue, guess we just copied it (duplicated it)
